### PR TITLE
fix help and focusdev from debug

### PR DESCRIPTION
**Pull request by enola-dkfz**

### CODE REVIEW

File: dev/focusdev [hunk 1]
```diff
@@ -55,7 +55,7 @@ function build() {
     BUILD_DOCKER=0
     BACK=$(pwd)
     cd $SD/..
-    FOCUS=./target/release/focus
+    FOCUS=./target/debug/focus
```
Comment (lablans) [inline dev/focusdev:58]: Why are you changing this?

File: dev/focusdev [hunk 2]
```diff
@@ -95,7 +95,7 @@ function start {
     pki/pki setup
     build $@
     docker compose up --no-build --no-recreate -d
-    docker compose logs focus -f
+    docker compose logs -f
```
Comment (lablans) [inline dev/focusdev:98]: same here

**Review by lablans — CHANGES_REQUESTED** [review]

**Review by TKussel — APPROVED** [review]
Assuming a debug build for local dev environment is sensible!

File: dev/focusdev (hunk 1 above)
Comment (Threated) [reply dev/focusdev:58]: This is changed because this is just for debugging in beamdev this is also debug

**Review by Threated — APPROVED** [review]
Fixes #60

File: dev/focusdev (hunk 2 above)
Comment (Threated) [reply dev/focusdev:98]: If you want more logs you can change it to `docker compose logs focus blaze -f`. Keep in mind if you want you can always `cd dev && docker compose logs -f` when you need to debug other stuff

Comment (enola-dkfz) [reply dev/focusdev:98]: I'm just going to revert it

**Review by lablans — APPROVED** [review]

**Pull request merged**